lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -480,16 +480,19 @@ def _section_course_info(course, access):
     return section_data
 
 
-def _section_membership(course, access, is_white_label):
+def _section_membership(course, access):
     """ Provide data for the corresponding dashboard section """
     course_key = course.id
     ccx_enabled = settings.FEATURES.get('CUSTOM_COURSES_EDX', False) and course.enable_ccx
+    default_enrollment_role_choices = ['Partner', 'Support', 'Learner']
```

Comment:
please reverse the order to Learner, Support, Partner.

Reply:
also should these default values be defined in a static somewhere so they are easy to find and modify if necessary